tests/hats_import/catalog/test_run_round_trip.py

```diff
@@ -66,6 +68,138 @@ def test_import_snappy_source_table(
     assert metadata.num_row_groups == 1
     assert metadata.row_group(0).column(0).compression == "SNAPPY"
 
+    # Check that the pixels have been sorted by the default _healpix_29 values
+    pixel_data = pq.read_table(output_file)
+    sorting_columns = metadata.row_group(0).sorting_columns
+    ordering_tuples = pq.SortingColumn.to_ordering(pixel_data.schema, sorting_columns)[0]
+    assert ordering_tuples[0] == (SPATIAL_INDEX_COLUMN, "ascending")
+    assert pixel_data.equals(pixel_data.sort_by(SPATIAL_INDEX_COLUMN))
+
+
+@pytest.mark.dask
+def test_import_with_num_row_groups(
+    dask_client,
+    small_sky_source_dir,
+    tmp_path,
+):
+    """The row group size will be specified using `row_group_kwargs`"""
+    args = ImportArguments(
+        output_artifact_name="small_sky_source_catalog.parquet",
+        input_path=small_sky_source_dir,
+        file_reader="csv",
+        catalog_type="source",
+        ra_column="source_ra",
+        dec_column="source_dec",
+        sort_columns="source_id",
+        output_path=tmp_path,
+        dask_tmp=tmp_path,
+        highest_healpix_order=2,
+        pixel_threshold=3_000,
+        progress_bar=False,
+        # Sneak in a test of custom row group size
+        row_group_kwargs={"num_rows": 100},
+    )
+
+    runner.run(args, dask_client)
+
+    # Check that the catalog metadata file exists
+    catalog = read_hats(args.catalog_path)
+    assert catalog.on_disk
+    assert catalog.catalog_path == args.catalog_path
+    assert catalog.catalog_info.ra_column == "source_ra"
+    assert catalog.catalog_info.dec_column == "source_dec"
+    assert len(catalog.get_healpix_pixels()) == 14
+
+    output_file = os.path.join(args.catalog_path, "dataset", "Norder=1", "Dir=0", "Npix=47.parquet")
+    pixel = pq.ParquetFile(output_file)
+    data = pixel.read()
+    assert len(pixel.read()) == 2395
+
+    # Check that the number of row groups is the one expected
+    metadata = pixel.metadata
+    assert metadata.num_rows == 2395
+    assert metadata.num_row_groups == math.ceil(metadata.num_rows / 100)
```

Comment:
```suggestion
    # 24 = math.ceil(metadata.num_rows / 100)
    assert metadata.num_row_groups == 24
```

I don't like doing math when I'm debugging.